iguana/iguana_notary.c

```diff
@@ -814,8 +814,11 @@ ZERO_ARGS(dpow,notarychains)
     return(jprint(array,1));
 }
 
+// FIXME Alright, unsure if anyone still uses this
+// safest option may be to remove it and see if anyone complains
 STRING_AND_INT(dpow,fundnotaries,symbol,numblocks)
 {
+    return(clonestr("{\"error\":\"Deprecated. Can be reenabled if needed. see Alright\"}"));
```

Comment:
do we want this?

Reply:
If you aren't using this, I'm not sure who would be. It is a method for funding all notaries at once. Last time I saw it used was by Adam.

Reply:
I'm not using it, got other methods